### ISSUE TITLE
Retain 1.7.10 R1/1.7.9 R2 support

### ISSUE THREAD

**Issue by johnfriedrich** [BuycraftPlugin/BuycraftX#6]
Hi,
I can't see any important reasons why support for 1.7.9/1.7.10 has been dropped completely.
Yes for sure, it's a lot easier to maintain just one version, but this is Buycraft. It shouldn't drop support for quite a big mass of customers.
18% of all minecraft servers are still on 1.7.10 and this will not change as these servers are spigot-Forge 1.7.10 servers.
Source:
http://mcstats.org/plugin/Vault (scroll down to Game versions)

At least the plugin should load it's core functionality on 1.7.10 and disable features like buy signs or shop gui, to let all customers benefit from the optimizations this new version brings.
This change would also lower support requests because users don't complain that the new version is not loading on their modded 1.7.10 server,

Besides from that, thanks for making  Buycraft more efficient!

**Comment by minecrafter** [comment]
With Minecraft 1.9 around the corner and 1.7 support dropped from Spigot, we felt that it made little sense to keep supporting old versions of Minecraft for what would be limited benefit overall.

Your argument with regards to Forge is both true and false. Most mods are just now updating to 1.8.9 (and will work with the Sponge port of BuycraftX once we have released it), but none of the "Spigot-Forge" (Cauldron) forks are updating.

I'm not going to close this issue as we'd like to gauge how much customer demand from a 1.7 release of BuycraftX we would get, but at present we lean towards keeping support for anything older than 1.8 dropped.

**tl;dr:** Maybe, but due to other events (Sponge nearing a release and mod updates to 1.8.x) we're leaning towards no.

Even if we choose to not support BuycraftX on 1.7.10, you could still fork this plugin to make it work (it's still open-source).

**Comment by LeeMcNeil** [comment]
I'd like to add that the old plugin (and the API it runs on) will be deprecated but still available to use until around January 2017. We feel that this gives plenty of time for the phase out of 1.7.9.

**Comment by minecrafter** [comment]
[Minecraft 1.9 release is one week away](https://mojang.com/2016/02/minecraft-19-pre-release-1/).

**Comment by minecrafter** [comment]
Closing this issue, since we won't be adding 1.7 support any time soon.

**Comment by minecrafter** [comment]
After the fact: 1.7.10 support has been readded. However, we will not be adding support for Bukkit ports aside from CraftBukkit, Spigot, and Paper. Cauldron and Thermos require too much work to support, which would be outweighted by the large amount of users using more common platforms. For modded servers, the Sponge port is more appropriate.
